branding/pdf/entities.adoc

```diff
@@ -47,6 +47,8 @@
 :sle: SUSE Linux Enterprise
 :sleses: SUSE Linux Enterprise Server with Expanded Support
 :es: Expanded Support
+:slemicro: SLE Micro
+:microos: MicroOS
```

Comment:
If you use `:slemicro:`, then use opensusemicroos: `openSUSE MicroOS`. I prefer this to have only `:micro: micro`